source/applications/advanced/mask_point_cloud.py

```diff
@@ -14,16 +14,19 @@
 from utils.paths import get_sample_data_path
 
 
-def _display_rgb(rgb):
+def _display_rgb(rgb, title):
     """Display RGB image.
 
     Args:
-        rgb: RGB image
+        rgb: RGB image (HxWx3 darray)
```

Comment:
Yes, this is the kind of docstring I want to see. No need for "OpenCV image" :)

Reply:
Btw it's "ndarray" or "Numpy array", not "darray".

Reply:
done.